djangocms_content_expiry/templates/djangocms_content_expiry/admin/icons/calendar_copy_icon.html

```diff
@@ -1,2 +1,2 @@
 {% load static i18n %}
-<a class="btn cms-moderation-action-btn js-moderation-action" id="copy_content_expiry_id_{{ field_id }}" href="{{ url }}" title="{% trans 'View Expiry' %}"><span class="svg-juxtaposed-font"><img src="{% static 'djangocms_content_expiry/svg/calendar_copy.svg' %}" /></span></a>
+<a class="btn cms-moderation-action-btn js-moderation-action" id="copy_content_expiry_id_{{ field_id }}" href="{{ url }}" title="{% trans 'View expiry & compliance number' %}"><span class="svg-juxtaposed-font"><img src="{% static 'djangocms_content_expiry/svg/calendar_copy.svg' %}" /></span></a>
```

Comment:
This one shouldn't actually be View Expiry, it is the force copy and paste action. Whilst we are changing this we should correct / fix it.

Reply:
As part of the ticket for introducing the burger menu into the versioning table, the label should be "Copy Expiry", so I have now changed it to this, can confirm in the standup tomorrow.